modules/store/src/store.ts

```diff
@@ -112,7 +112,9 @@ export class Store<T = object>
     selector: (state: T) => K,
     options?: SelectSignalOptions<K>
   ): Signal<K> {
-    return computed(() => selector(this.state()), { equal: options?.equal });
+    return computed(() => selector(this.state()), {
+      equal: options?.equal || ((previous, current) => previous === current),
+    });
```

Comment:
LGTM 👍 

Can you add a test to the `integration_signals.spec.ts` to verify this fix?